sp800-63-3/cover.md

```diff
@@ -91,14 +91,14 @@ http://dx.doi.org/10.6028/NIST.SP.XXX
 
 {::comment}
 
-**Comments on this publication may be submitted to eauth-comments@nist.gov  
+**Comments on this publication may be submitted to eauth-comment@nist.gov  
```

Comment:
Do we want a new email address? We can forward the eauth-comment addy if we want to retain that one.